missions/2PzD_Template_v3_0.VR/customization/geardefs/British/gearDefUK.sqf

```diff
@@ -14,7 +14,7 @@
 
 //Backpack
 
-#define UK_BPK_Radio                "fow_b_us_radio"
+#define UK_BPK_Radio                "B_LIB_US_Radio"
 #define UK_BPG_Radio                "fow_b_uk_p37_radio_blanco"
```

Comment:
I'm thinking we can just remove these two. the Brits use the US radios and those US files get loaded anyway, so those definitions should always exist.